src/main/lombok/org/heigit/ohsome/ohsomeapi/executor/ContributionsExecutor.java

```diff
@@ -32,22 +44,44 @@ public ContributionsExecutor(HttpServletRequest servletRequest,
   }
 
   /**
-   * Performs a count calculation using contributions for the /contributions/count or the
-   * /users/count endpoint.
+   * Handler method for count calculation of the endpoints /contributions/count,
+   * /contributions/density, /contribution/latest/count or /users/count.
    *
-   * @return {@link org.heigit.ohsome.ohsomeapi.output.Response Response}
+   * @param isUsersRequest the boolean value relative to the endpoint /users/count
+   * @param isContributionsLatestCount the boolean value relative to the endpoint
+   *        /contributions/latest
+   * @return DefaultAggregationResponse {@link org.heigit.ohsome.ohsomeapi.output.Response Response}
    * @throws Exception thrown by
    *         {@link org.heigit.ohsome.ohsomeapi.inputprocessing.InputProcessor#processParameters()
    *         processParameters},
+   *         {@link org.heigit.ohsome.ohsomeapi.inputprocessing.InputProcessor
+   *         #processParameters(ComputeMode) processParameters} and
    *         {@link org.heigit.bigspatialdata.oshdb.api.mapreducer.MapAggregator#count() count}
+   * @throws UnsupportedOperationException thrown by
+   *         {@link org.heigit.ohsome.ohsomeapi.executor.ContributionsExecutor
+   *         #usersCount(MapReducer) usersCount} and
+   *         {@link org.heigit.ohsome.ohsomeapi.executor.ContributionsExecutor
+   *         #contributionsCount(MapReducer, boolean) contributionsCount}
    */
-  public Response count(boolean isUsersRequest) throws Exception {
-    MapReducer<OSMContribution> mapRed = inputProcessor.processParameters();
+  public Response count(boolean isUsersRequest, boolean isContributionsLatestCount)
+      throws UnsupportedOperationException, Exception {
+    MapReducer<OSMContribution> mapRed;
     final SortedMap<OSHDBTimestamp, ? extends Number> result;
+    if (isContributionsLatestCount) {
+      inputProcessor.getProcessingData().setFullHistory(true);
```

Comment:
why is this necessary (only for the _latest_ resource)? IMHO it's not obvious, so it should be documented here with an inline comment.

Reply:
Regarding this point:
1. I think the Inputprocessor is a little bit outdated. With the whole bunch of endpoints that we have by now, the check of `isFullHistory()` (and some other checks) is not intuitive. Some future refactoring would improve the readability.
2. The documentation for groupByEntity() in the oshdb API refers only to map() and flatMap(). It was not intuitive that also calling filter() prevents the call of groupByEntity(). Is there some room for docs improvement there?

Reply:
ad 2.: yes, you are right. some filters also act like a `flatMap` in the OSHDB. This is indeed missing/outdated information in the OSHDB docs. Can you [open a ticket](https://github.com/giscience/oshdb/issues/new) for that in the OSHDB repo? Thanks!